src/strategies/cream/index.ts

```diff
@@ -0,0 +1,144 @@
+import { formatUnits } from '@ethersproject/units';
+import { getAddress } from '@ethersproject/address';
+import { subgraphRequest, multicall } from '../../utils';
+
+export const author = 'bun919tw';
+export const version = '0.1.0';
+
+const CREAM_SWAP_SUBGRAPH_URL = {
+  1: 'https://api.thegraph.com/subgraphs/name/creamfinancedev/cream-swap-v2',
+  3: 'https://api.thegraph.com/subgraphs/name/creamfinancedev/cream-swap-dev'
+};
+
+const longTermPoolABI = [
+  {
+    constant: true,
+    inputs: [
+      {
+        internalType: "address",
+        name: "account",
+        type: "address"
+      }
+    ],
+    name: "balanceOf",
+    outputs: [
+      {
+        internalType: "uint256",
+        name: "",
+        type: "uint256"
+      }
+    ],
+    payable: false,
+    stateMutability: "view",
+    type: "function"
+  }
+];
+
+export async function strategy(
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+) {
+
+  const creamAddress = options.pools.find(pool => pool.name === 'CREAM').address;
+  let score = await creamSwapScore(network, addresses, creamAddress, snapshot);
```

Comment:
Can you make the 2 requests happen in parallel with something like this:
```js
let [score, creamBalance] = await Promise.all([
  creamSwapScore(network, addresses, creamAddress, snapshot),
  creamBalanceOf(network, provider, addresses, options, snapshot)
]);
```
This will be faster to resolve than if we process request one by one

Reply:
Good call, will do that!